combat-trainer.lic

```diff
@@ -189,12 +191,46 @@ class LootProcess
   def dispose_body(game_state)
     return if DRRoom.dead_npcs.length < 1
     game_state['mob_died'] = true
-
-    check_skinning(DRRoom.dead_npcs.first) if @skin
-
+      if @necro
```

Comment:
Short answer: yes.

```combat_vars.blah``` is a shortcut to get you whatever is stored in the YAML (a top-level var with name 'blah'). It only works on the top-level settings - anything below that has to be accessed differently.

The easy case is when the top-level name in the YAML maps directly to a value. For example, ```cambrinth: armband``` would be accessed with ```combat_vars.cambrinth``` and return a string, ```armband```.

Skinning is slightly more complicated. Given a YAML of:
```
skinning:
  skin: true
  arrange_all: false
  arrange_count: 0
```
you could say ```combat_vars.skinning``` and it parses everything under the name ```skinning```. In this case, the data under name ```skinning``` are a bunch of key-value pairs. In Ruby-speak, that means a Hash: ```{ 'skin' => true, 'arrange_all' => false, 'arrange_count' => 0 }```. To get something out of the hash, access it by a key: ```combat_vars.skinning['skin']``` would give you ```true```.